fast64_internal/f3d/f3d_material.py

```diff
@@ -2100,10 +2100,6 @@ def load_handler(dummy):
 
 bpy.app.handlers.load_post.append(load_handler)
 
-# bpy.context.mode returns the keys here, while the values are required by bpy.ops.object.mode_set
-BLENDER_MODE_TO_MODE_SET = {"PAINT_VERTEX": "VERTEX_PAINT", "EDIT_MESH": "EDIT"}
-get_mode_set_from_context_mode = lambda mode: BLENDER_MODE_TO_MODE_SET.get(mode, "OBJECT")
```

Comment:
you moved this but didn't import it in this file

Reply:
`from ..utility import *`

Reply:
:+1: 
star imports my nemesis